examples/todo-typescript/src/server/actions.ts

```diff
@@ -0,0 +1,33 @@
+import HttpError from '@wasp/core/HttpError.js';
+import { Context, Task } from './serverTypes'
+
+type createArgs = { description: Task['description'] };
```

Comment:
Nice work typing these!

Heads up, TypeScript uses capital letters for type names, which means this one should be called `CreateArgs`.

The same applies for the rest of the PR.